apps/admin/traefik2/ptl-intsvc/00.yaml

```diff
@@ -4,6 +4,15 @@ metadata:
   name: traefik2
   namespace: admin
 spec:
+  chart:
```

Comment:
I would do ptl in sds first just to be sure of Jenkins working. Should be fine but best to keep it consistent

Reply:
ah -sure, I was thinking this would be different then ptl  environment as it said ptl-intsvc :)

Reply:
yeah, sadly they're not the same across both repos but they are the same use case

Reply:
sure